modules/imaging_qc/php/imaging_qc.class.inc

```diff
@@ -386,5 +389,29 @@ class Imaging_QC extends \NDB_Menu_Filter
             )
         );
     }
-}
 
+    /**
+     * This method overrides the NBD_Page handle function.
+     * It checks if the MRI Parameter Form table exists
+     * before calling the parent handle function.
+     *
+     * @param ServerRequestInterface $request The PSR15 Request being handled
+     *
+     * @return ResponseInterface The PSR15 response for the page.
+     */
+    public function handle(ServerRequestInterface $request) : ResponseInterface
+    {
+        $db = $this->loris->getDatabaseConnection();
+        if ($db->tableExists('mri_parameter_form')) {
+            return parent::handle($request);
+        } else {
+            return (new \LORIS\Http\Response())
```

Comment:
@driusan I am getting this error in the tests because of this line, do you know how to fix it?
`PhanUndeclaredMethod Call to undeclared method \Psr\Http\Message\MessageInterface`
It works when running it locally :/